src/WebdriverClassicDriver.php

```diff
@@ -1128,19 +1034,6 @@ private function ensureInputType(
         }
     }
 
-    /**
-     * @throws DriverException
-     */
-    private function trigger(
-        #[Language('XPath')]
-        string $xpath,
-        string $event,
-        #[Language('JSON')]
-        string $options = '{}'
-    ): void {
-        $this->withSyn()->executeJsOnXpath($xpath, "window.syn.trigger(arguments[0], '$event', $options)");
```

Comment:
Why removing it completely. Can't fire events using native JavaScript like so:

```js
$0.dispatchEvent(new Event('blur'));
```

Reply:
That's the whole point: we should not only fire a fake event. That's not the intended behavior of this method. `blur` is expected to actually remove the focus from the element.

But this case should probably be doable (see my previous review)

Reply:
> Why removing it completely? Can't fire events using native JavaScript like so:
> 
> ```js
> $0.dispatchEvent(new Event('blur'));
> ```

@aik099 

I've checked the [original PR](https://github.com/uuf6429/webdriver-classic-driver/pull/2#issuecomment-1742088874) and the reason (from my side) was that end users cannot directly focus or blur elements. They perform actions that focus/blur as a side-effect. For example, when you "tab" between fields or "click" fields, you are firstly interacting with the keyboard or mouse. There isn't a way for a user to "just" focus or blur something.

And since we were removing syn and some related events, it made sense to me to also disable this functionality (if mink also eventually goes in that direction).

But I don't have a strong opinion on this, I'm also fine adding it back (as I just did).

Reply:
This has been reimplemented back. Let's continue the other discussion in the other comment thread.